File: exercices.py
# thermometre et change

# def thermometer(celsius):
#     return (celsius *9/5)+32

# celsius = 18.5
# print(str(celsius)+ "degrees Celsius are" + str(thermometer(21.5))+ "degrees Farehneit")


# def change(eur):
#     text1 = "the equivalent of "
#     text2 = 'euros in swiss francs is' 
#     return text1 + str(chf) + text2 + str(round((chf/1.1),2))

# chf = 4000
# print(change(chf))

# inputs

# print("please enter your name")
# name = input()

# print("your name is" +name)

# name = input("Please enter your name: ")
# print("Your name is " + name)

# x = input("Enter a number: ")
# y = input("Enter a second number: ")
# result = int(x) + int(y)

# message = "The result of " + x + " plus " + y + " is " + str(result)

# print(message)


print("enter a temperature in celsius")

def temp(input):
    return (input*9/5)+32


celsius = float(input())

print("the equivalent in F is : " + str(temp(celsius)))

File: exercices.js

function thermo(celsius){
    
    var message1 = "degrees celius are"
    var far = (celsius*5/9)+32
    return r.toString() + message1 + far.toString()
}   

var r = 18.5;

console.log(thermo(r))




function change(eur){
    txt1 = "eur is the equivalent of"
    txt2 = Math.round(eur/1.1),2;
    return x.toString() + txt1 + txt2;
}

var x = 200;

console.log(change(x));


var apples = 7
var pear = 7

if (apples => pear) {
    console.log("there are more or equal apples than pears")
}


var secret = "raffaele"
var guess = prompt("Guess the word : ")

if (secret == guess) {
    console.log("Congrats!")
}

else {
    console.log("Try again")
}



var weather = "sunny"

if (weather == "sunny"){
    console.log("take sunglasses");
}
else if (weather == "rainy"){
    console.log("take umbrella");
}
else if (weather == "cold"){
    console.log("take jacket");
}
else {
    console.log("idk");
}




function greet(gender){

    var msgmale = "Hello Sir !"
    var msgfemale = "Hello Madam!"
    var msgneutral = "Hello!"

    if (gender == "male"){
        return msgmale;
        } 
    else if (gender == "female"){
        return msgfemale;
        }
    else{
        return msgneutral;
    }
}

    var input = "male"

    console.log(greet(input))

var nb1 = 13
var nb2 = 789

if (Number(nb1) > 10 && Number(nb2) > 10){
    console.log("both are greater than 10")
}
else {
    console.log("at least one is less than 10")
}


var a = 8;
var b = 12;

if (a > 10 || b > 10) {
  console.log("At least one of the numbers is greater than 10");
} else {
  console.log("Both numbers are not greater than 10");
}




var lang = "Python"
var laang = "Java"
var nb = 10
var color = "blue"

if (lang=="Python" || lang=="Java"){
    console.log("ok");
}

if (lang=="Python" && laang=="Java"){
    console.log("TCC");
}




var lang = "Python"
var laang = "Java"
var nb = 10
var color = "blue"

if (lang=="Python" || lang=="Java"){
    console.log("ok");
}

if (lang=="Python" && laang=="Java"){
    console.log("TCC");
}

